src/diagnostics/LogPointsManager.ts

```diff
@@ -88,7 +88,7 @@ class DebugSessionManager {
         }
 
         const result: ISetLogpointResponse = await this._debugSession.customRequest("setLogpoint", { scriptId, lineNumber, columnNumber, expression });
-        if (result.error) {
+        if (result.error !== undefined) {
```

Comment:
Why? Couldn't result.error be null?

Reply:
I'm not familiar with this code. Do you know if result.error can be null, and what is the semantics of that?

Reply:
It depends on what the server responds with. It's purely modeling the REST API response payload. From the perspective of VS Code, there is not point of making any assumption. The semantic is that as long as result.error is non-empty, it's an error. 

Suggest revert this change.

Reply:
I added this because the request was failing with error = "" so we weren't seeing any error messages because the if evaluated to false.
Changing this to: if (result.error !== undefined && result.error !== null) {

Reply:
Sounds good.